src/richie/apps/search/filter_definitions/base.py

```diff
@@ -252,8 +252,7 @@ def get_faceted_definitions(self, facets, data, *args, **kwargs):
         Add the counts to the values from the initial definition to make them complete
         and sorted as the frontend expects them.
         """
-        # Get filter values to derive human names
-        values = self.get_values()
+        static_values = self.get_values()
```

Comment:
should be called get_human_names since you later do 
human_name = static_values[name]